squidb-processor/src/com/yahoo/squidb/processor/plugins/defaults/AndroidModelPlugin.java

```diff
@@ -61,6 +61,12 @@ public AndroidModelPlugin(ModelSpec<?> modelSpec, PluginEnvironment pluginEnv) {
     }
 
     @Override
+    public List<String> getSupportedOptions() {
```

Comment:
All these default plugins don't need to implement this method because all the built-in default options are explicitly added to the set above.